docker/start_server.sh

```diff
@@ -8,6 +8,8 @@ function symlink_client_files() {
     ln -s /client/client/res/names/ /app/res/names
     ln -s /client/client/res/CDServer.sqlite /app/res/CDServer.sqlite
     ln -s /client/client/locale/locale.xml /app/locale/locale.xml
+    # need to create this file so the server knows the client is unpacked (see `dCommon/dClient/AssetManager.cpp`)
```

Comment:
why? this forces docker to only used unpacked client assets, when it should be able to use a packed client with no issues.

Reply:
Haven't done much with unpacking in a while, but if I remember correctly [this script](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/docker/setup.sh) is unpacking the client if it hasn't been already. So from the server's perspective in Docker, the client will always be unpacked.